Add explicit clarification that values like 0 and "" are truthy in Elixir (#13300)
elixir-lang/elixir: lib/elixir/pages/getting-started/basic-types.md

true
```

> Since only `false` and `nil` are considered "falsy", values like `0` and `""` that some other programming languages consider to also be "falsy" are in fact "truthy" in Elixir.

Reviewer: We already mention false and nil right before, so what about this:

```suggestion
Similarly, values like `0` and `""`, which some other programming languages consider to also be "falsy", are also "truthy" in Elixir.
```

Author: Sounds good.
